app/src/main/java/org/sopt/official/feature/home/model/AppServiceState.kt

```diff
@@ -0,0 +1,11 @@
+package org.sopt.official.feature.home.model
+
+data class AppServiceState(
+    val showSoptamp: Boolean = false,
+    val showPoke: Boolean = false
+)
```

Comment:
```suggestion
data class AppServiceUiState(
    val showSoptamp: Boolean = false,
    val showPoke: Boolean = false
)
```

어떤 State인지 잘 모를수 있으니...!!

Reply:
아 맞네요!! UI 관련된 코드니 네이밍 수정하겠습니다.